Dockerfile

```diff
@@ -8,6 +8,10 @@ RUN rm -rf ./*
 # Make sure the directory for individual app logs exists
 RUN mkdir -p /var/log/shiny-server
 
+# Install apt-get dependecies
+ADD apt-dependencies apt-dependencies
```

Comment:
This will fail if there isn't an apt-dependencies file, so the next line can't use a conditional - these two lines should probably just be present or commented out